lib/rucio/rse/protocols/gfal.py

```diff
@@ -176,15 +182,20 @@ def connect(self):
         """
         self.logger(logging.DEBUG, 'connecting to storage')
 
+        self.__ctx = gfal2.creat_context()  # pylint: disable=no-member
+        self.__ctx.set_opt_string_list("SRM PLUGIN", "TURL_PROTOCOLS", ["gsiftp", "rfio", "gsidcap", "dcap", "kdcap"])
```

Comment:
this is missing https and davs

Reply:
(I know, it was wrong even before :-)

Reply:
https;gsiftp;root ? to be coherent with TURL_PROTOCOLS defined here https://fts3-atlas.cern.ch:8449/fts3/ftsmon/#/config/gfal2

Reply:
@mpatrascoiu is the order of protocols here important for gfal/davix?

Reply:
Is this conversation resolved?